### ISSUE TITLE
Feature request: add "additional parameters" argument when using sm_forcertd

### ISSUE THREAD

**Issue by Tiagoquix** [Phil25/RTD#87]
The new RTD major update with its default values brought a lot of cool things, but, at the same time, it ended up removing certain aspects of the Old RTD™.

I would like to create a new argument when using `sm_forcertd` — `RTD mode`:

`[SM] Usage: sm_forcertd <player> <perk id>* <time>* <override class restriction (0 / 1)>* <additional parameters>`

This argument would cause the effect specified via ForceRTD to use a different mode/attribute etc. than the configured defaults.

The argument, of course, would be optional. If none was given, the default behavior would be assumed.

For instance:
`sm_forcertd @me 4 60 1 mode;1` -> would make myself have the Flying effect, but with the Noclip mode.

`sm_forcertd @me 0 60 1 mode;1;uber;1` -> godmode with normal self-damage and uber (overriding default values); use default "resistance" value from server config.

`sm_forcertd @me 0 60 1 mode;-1;resistance;0.75` -> no self-damage, use default "uber" value in server's configuration file, and 25% resistance.

`sm_forcertd @me 35 10 1 force;8192` -> Firework with double force.

And so on.

Basically, the idea is to be able to customize anything inside the `settings` section from individual effects directly from the command.

It would be cool to have this feature implemented.

Thanks for reading.

**Comment by Phil25** [comment]
Hey thanks for the issue! I actually had this idea already and planning to add it to the future 3.0.0 version.

I’m gonna close this as a duplicate of #76.
